articles/azure-monitor/app/api-filtering-sampling.md

```diff
@@ -195,7 +195,30 @@ public void Process(ITelemetry item)
 
 <a name="add-properties"></a>
 
-## Add properties: ITelemetryInitializer
+### JavaScript Web applications
+
+**Filtering using ITelemetryInitializer**
+
+1. Create a telemetry initializer callback function.  The callback function takes an ITelemetryItem as a parameter, which is the event that is being processed.  Returning false from this callback will result in the telemetry item to be filtered out.  
+
+```JS
+var filteringFunction = (envelope) => {
+  if (envelope.data.someField == 'tobefilteredout') {
+  	return false;
+  }
+  
+  return true;
```

Comment:
What do you think of this just being
```js
return (envelope.data.someField === 'tobefilteredout')
```

Reply:
I love it, but want the example to be verbose.